File: id/src/main/java/com/simprints/id/tools/Vibrate.java
package com.simprints.id.tools;

import android.content.Context;
import android.os.Vibrator;

public class Vibrate {
    public static void vibrate(Context context) {
        ((Vibrator) context.getSystemService(Context.VIBRATOR_SERVICE)).vibrate((long) 100);
    }
}

File: id/src/main/java/com/simprints/id/tools/Vibrate.kt
package com.simprints.id.tools

import android.content.Context
import android.os.Vibrator

object Vibrate {
    fun vibrate(context: Context) {
        (context.getSystemService(Context.VIBRATOR_SERVICE) as Vibrator).vibrate(100.toLong())
    }
}
